roborazzi-junit5/src/main/kotlin/com/github/takahirom/roborazzi/junit5/CurrentTestInfo.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.takahirom.roborazzi.junit5
+
+import org.junit.jupiter.api.TestInfo
+
+/**
+ * A shared, static data container for storing the currently executed test method.
+ * This is updated by [RoborazziExtension] and read by [JUnit5TestNameExtractionStrategy]
+ * from different class loaders, bridging the gap between test definition and their execution.
+ */
+internal object CurrentTestInfo {
+  private val concurrentRef = ThreadLocal<TestInfo>()
```

Comment:
I wasn't aware that we needed to use ThreadLocal. I'm a bit concerned that we are currently using object for RoborazziContext, and if we switch to ThreadLocal, it might break some tests that use a different thread. I think this issue isn't directly related to this PR, but we should consider it later.

Reply:
It's a bit of a premature optimization at this point, because we haven't figured out parallel test execution for Robolectric yet (and I suppose that limitation extends to Roborazzi). But in the future, it's theoretically possible for multiple tests to run concurrently on different threads, and every test should be able to store its own `TestInfo` here